v4.3.0 Changelog Update (#3415)
kyb3r/modmail: CHANGELOG.md

however, insignificant breaking changes do not guarantee a major version bump, see the reasoning [here](https://github.com/modmail-dev/modmail/issues/319). If you're a plugin developer, note the "BREAKING" section.

# v4.3.0

### Added
- Ability to rename snippets and aliases [#3383](https://github.com/modmail-dev/Modmail/pull/3412)
- Bot events when threads is snoozed/unsnoozed [#3412](https://github.com/modmail-dev/Modmail/pull/3412)
- Check for threadmenu and advancedmenu being enabled concurrently. [#3411](https://github.com/modmail-dev/Modmail/pull/3411)
- Check for git installation before attempting autoupdate or update command. [#3406](https://github.com/modmail-dev/Modmail/pull/3406)
- Arguments in responses, similar to snippets, but can be used anywhere. [#3418](https://github.com/modmail-dev/Modmail/pull/3418)
- Check to see if bot is using Git repository for install before running update or autoupdate. [#3406](https://github.com/modmail-dev/Modmail/pull/3406)
- Ability to add attachments directly to snippets via storing image in MongoDB. [#3421](https://github.com/modmail-dev/Modmail/pull/3421)

### Changed

> - Updated details on v4.2.1 changelog to better explain changes [#3409](https://github.com/modmail-dev/Modmail/pull/3409)

Reviewer: This line is pointing to an irrelevant PR that is closed. I think it might be the wrong link?

> - Updated plugin registry [#3409](https://github.com/modmail-dev/Modmail/pull/3409)

Reviewer: This PR is closed. Did this get changed in another PR potentially?

This project mostly adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html);
however, insignificant breaking changes do not guarantee a major version bump, see the reasoning [here](https://github.com/modmail-dev/modmail/issues/319). If you're a plugin developer, note the "BREAKING" section.

# v4.3.0

### Added
- Ability to rename snippets and aliases [#3383](https://github.com/modmail-dev/Modmail/pull/3412)
- Bot events when threads is snoozed/unsnoozed [#3412](https://github.com/modmail-dev/Modmail/pull/3412)
- Check for threadmenu and advancedmenu being enabled concurrently. [#3411](https://github.com/modmail-dev/Modmail/pull/3411)
- Check for git installation before attempting autoupdate or update command. [#3406](https://github.com/modmail-dev/Modmail/pull/3406)

> - Check to see if bot is using Git repository for install before running update or autoupdate. [#3406](https://github.com/modmail-dev/Modmail/pull/3406)

Reviewer 2: ```suggestion
- Check for git installation before attempting autoupdate or update command. [#3406](https://github.com/modmail-dev/Modmail/pull/3406)
```
Probably just added twice by accident. Just one needed ig.
